### DISCUSSION TITLE
[Help]: slides in Embla options only accepts HTMLElements/ NodeList/ string

### DISCUSSION THREAD

**Discussion by bansal1789** [davidjerleke/embla-carousel#767] [Help]
### Summary

I want to pass JSX.Element[] to the slides in the options for Embla Carousel. 

### If applicable, which variants of Embla Carousel are relevant to this question?

- [ ] embla-carousel **(Core)**
- [X] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by davidjerleke** [comment]
@bansal1789 thanks for your question.

I'm not sure why you would want to do that. What's the use case? What are you trying to achieve?
